openid-4-verifiable-presentations-1_0.md

```diff
@@ -1894,6 +1894,7 @@ The technology described in this specification was made available from contribut
 
    * added references to ISO/IEC 23220 and 18013 documents 
    * added `post` request method for Request URI
+   * Added IETF SD-JWT VC example
```

Comment:
Although it is in a section title examples (there's an issue about that...) I think it's better calling this a profile as it has normative requirements:

```suggestion
   * Added IETF SD-JWT VC profile
```

Reply:
Had a similar thought here fwiw